### ISSUE TITLE
Incorrect reading order with VoiceOver on 

### ISSUE THREAD

**Issue by qanitah** [cds-snc/covid-shield-mobile#175]
While VoiceOver is on and the user keeps flicking right to go item by item, some order is not correct on the first 4 screens of the app.

- after tab 2, right flick takes the user back to tab 1
- after tab 4, user goes to tab 3

The user should be able to go on all screens in order and all content should be announced.

iOS 13.5.1 on iPhone 7

**Comment by JuliannaR** [comment]
This was addressed in V11.
